specification/blueprint/resource-manager/Microsoft.Blueprint/preview/2017-11-11-preview/blueprint.json

```diff
@@ -0,0 +1,1707 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "BlueprintClient",
+    "description": "Azure Blueprint Client.",
+    "version": "2017-11-11-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/providers/{managementGroupsNamespace}/managementGroups/{managementGroupName}/providers/Microsoft.Blueprint/Blueprints/{blueprintName}": {
+      "put": {
+        "tags": [
+          "Blueprint"
+        ],
+        "operationId": "Blueprints_CreateOrUpdate",
+        "description": "Create or update Blueprint definition.",
+        "x-ms-examples": {
+          "Blueprint": {
+            "$ref": "./examples/Blueprint_Create.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagementGroupsNamespaceParameter"
+          },
+          {
+            "$ref": "#/parameters/ManagementGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/BlueprintNameParameter"
+          },
+          {
+            "name": "blueprint",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Blueprint"
+            },
+            "description": "Blueprint definition."
+          }
+        ],
+        "responses": {
+          "201": {
```

Comment:
we only return 201 for put request, no matter create or update.

Reply:
This would be good for consistency with other azure services. so 1st put returns 201 created
and then subsequent ones on the same resource return 200 ok

Reply:
Not making a blocking comment. Please address in the next api-version.